clients/da-vinci-client/src/main/java/com/linkedin/davinci/ingestion/DefaultIngestionBackend.java

```diff
@@ -79,6 +80,7 @@ public void startConsumption(VeniceStoreVersionConfig storeConfig, int partition
     if (!storeAndVersion.getFirst().isBlobTransferEnabled() || blobTransferManager == null) {
       runnable.run();
     } else {
+      storageService.openStore(storeConfig, svsSupplier);
```

Comment:
@FelixGV 
I saw you modified this interface to pass `svsSupplier` in the past:
https://github.com/linkedin/venice/pull/44

For the usage in this class, it is a little confusing:
```
Supplier<StoreVersionState> svsSupplier = () -> storageMetadataService.getStoreVersionState(storeVersion);
    syncStoreVersionConfig(storeAndVersion.getFirst(), storeConfig);

    Runnable runnable = () -> {
      AbstractStorageEngine storageEngine =
          storageService.openStoreForNewPartition(storeConfig, partition, svsSupplier);
```

So it first looks up SVS state from the metadata partition and then pass it to `StorageService#openStoreForNewPartition`, I wondered whether it is necessary to use this pattern as internally if `versionStateCache` is `null` in the get path, it will try to do the lookup internally.

My main question is that whether this param: `svsSupplier` is necessary or not in `openStore` and `openStoreForNewPartition`.

Reply:
I guess the user can pass a `null` supplier for such usage, right?